src/celeritas/ext/VecgeomTrackView.hh

```diff
@@ -230,6 +231,16 @@ CELER_FUNCTION VolumeId VecgeomTrackView::volume_id() const
     return VolumeId{this->volume().id()};
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Get the physical volume ID in the current cell.
+ */
+CELER_FUNCTION VolumeId VecgeomTrackView::volume_physid() const
+{
+    if (this->is_outside()) return VolumeId{0};  // outside
```

Comment:
Make the behavior of this analogous to the `volume_id` call:
```suggestion
    CELER_EXPECT(!this->is_outside());
```
Since might there not be a physical volume id `0` in the geometry?

Reply:
That was my first version, but I need it to return some int for points outside as well.
I will take another look into the callee code.

Reply:
Done as suggested.